packages/bot-strategy-runner/src/index.ts

```diff
@@ -118,13 +118,18 @@ const processExecutionOptions = async () => {
     .option("-fc, --fileConfig <path>", "input path to JSON config file.")
     .option("-uc, --urlConfig <path>", "url to JSON config hosted online. Private resources use access token")
     .option("-at, --accessToken <string>", "access token to access private configs online. EG private a repo")
+    .option("-env, --envConfig <bool>", "use a stringified config within your env set under `RUNNER_CONFIG")
     .parse(process.argv)
     .opts();
 
-  assert(options.fileConfig != undefined || options.urlConfig != undefined, "provide a file config or a URL config");
+  assert(
+    options.fileConfig != undefined || options.urlConfig != undefined || options.envConfig,
```

Comment:
why not write this as `assert(options.fileConfig || options.urlConfig || options.envConfig)`?

Reply:
fair! have refactored